### PR TITLE
Remove the upload and memory setting

### PR DESCRIPTION
**Pull request by nickvergessen**
Close #13990 

See #13990 for details and discussion

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Review by rcdailey — APPROVED** [review]
To the best of my limited knowledge of the code base, looks good to me. And some of this looks familiar because on my branch I started removing some of this. Looks like you covered a lot more than  I thought to look at though. Thanks so much for your change!

**Comment by MorrisJobke** [comment]
### Status of [16565](https://drone.nextcloud.com/nextcloud/server/16565): failure

 * ENABLE_REDIS=true, TESTS=nodb-codecov
<details><summary>Show full log</summary>

```
There was 1 error:

1) Test\IntegrityCheck\CheckerTest::testWriteCoreSignatureWithValidModifiedHtaccessAndUserIni
Exception: /drone/src/github.com/nextcloud/server/tests/data/integritycheck/htaccessWithValidModifiedContent/core is not writable

/drone/src/github.com/nextcloud/server/lib/private/IntegrityCheck/Checker.php:283
/drone/src/github.com/nextcloud/server/tests/lib/IntegrityCheck/CheckerTest.php:660

--

There was 1 failure:

1) Test\IntegrityCheck\CheckerTest::testVerifyCoreSignatureWithValidModifiedHtaccessAndUserIniSignatureData
Failed asserting that Array &0 (
    'INVALID_HASH' => Array &1 (
        '.htaccess' => Array &2 (
            'expected' => '9a37a508ad4cc3a7ff668e3cd63177891e5601143fa18bb605346020d5f3cd7979748beab956554aca43ae59ff146755cfe83de0f93a3a7bb328b1395b2ccf2f'
            'current' => 'e7140c092ecf6ed3401f922efc81b635cb253d90f56cad5ad9765511edf6b2d8c663fcb5648617d561ecd2cf7ae8503d7d2ce3b88f46844d5771909aa0dba188'
        )
        '.user.ini' => Array &3 (
            'expected' => '467d4f028c447895716a2b7859ed6e569f8ee34b87b51d73dab2e6a9ca4fbe63172c7be3e365ae864a60408286afcce852dd38ee544b61685ed4ee5e021fecb0'
            'current' => '8362e2f91924dde144162fe6b6757d88b8e6cd7266fd0e90e32aee37e662b7854384a211ac44e8703fa093833df6d5cceef5426eb141dc2638ff88d245cea3c0'
        )
    )
) is identical to Array &0 ().

/drone/src/github.com/nextcloud/server/tests/lib/IntegrityCheck/CheckerTest.php:763

```
</details>


 * DB=NODB, ENABLE_REDIS=true, PHP=7.1
<details><summary>Show full log</summary>

```
There was 1 error:

1) Test\IntegrityCheck\CheckerTest::testWriteCoreSignatureWithValidModifiedHtaccessAndUserIni
Exception: /drone/src/github.com/nextcloud/server/tests/data/integritycheck/htaccessWithValidModifiedContent/core is not writable

/drone/src/github.com/nextcloud/server/lib/private/IntegrityCheck/Checker.php:283
/drone/src/github.com/nextcloud/server/tests/lib/IntegrityCheck/CheckerTest.php:660

--

There was 1 failure:

1) Test\IntegrityCheck\CheckerTest::testVerifyCoreSignatureWithValidModifiedHtaccessAndUserIniSignatureData
Failed asserting that Array &0 (
    'INVALID_HASH' => Array &1 (
        '.htaccess' => Array &2 (
            'expected' => '9a37a508ad4cc3a7ff668e3cd63177891e5601143fa18bb605346020d5f3cd7979748beab956554aca43ae59ff146755cfe83de0f93a3a7bb328b1395b2ccf2f'
            'current' => 'e7140c092ecf6ed3401f922efc81b635cb253d90f56cad5ad9765511edf6b2d8c663fcb5648617d561ecd2cf7ae8503d7d2ce3b88f46844d5771909aa0dba188'
        )
        '.user.ini' => Array &3 (
            'expected' => '467d4f028c447895716a2b7859ed6e569f8ee34b87b51d73dab2e6a9ca4fbe63172c7be3e365ae864a60408286afcce852dd38ee544b61685ed4ee5e021fecb0'
            'current' => '8362e2f91924dde144162fe6b6757d88b8e6cd7266fd0e90e32aee37e662b7854384a211ac44e8703fa093833df6d5cceef5426eb141dc2638ff88d245cea3c0'
        )
    )
) is identical to Array &0 ().

/drone/src/github.com/nextcloud/server/tests/lib/IntegrityCheck/CheckerTest.php:763

```
</details>


 * DB=NODB, ENABLE_REDIS=false, PHP=7.2
<details><summary>Show full log</summary>

```
There was 1 error:

1) Test\IntegrityCheck\CheckerTest::testWriteCoreSignatureWithValidModifiedHtaccessAndUserIni
Exception: /drone/src/github.com/nextcloud/server/tests/data/integritycheck/htaccessWithValidModifiedContent/core is not writable

/drone/src/github.com/nextcloud/server/lib/private/IntegrityCheck/Checker.php:283
/drone/src/github.com/nextcloud/server/tests/lib/IntegrityCheck/CheckerTest.php:660

--

There was 1 failure:

1) Test\IntegrityCheck\CheckerTest::testVerifyCoreSignatureWithValidModifiedHtaccessAndUserIniSignatureData
Failed asserting that Array &0 (
    'INVALID_HASH' => Array &1 (
        '.htaccess' => Array &2 (
            'expected' => '9a37a508ad4cc3a7ff668e3cd63177891e5601143fa18bb605346020d5f3cd7979748beab956554aca43ae59ff146755cfe83de0f93a3a7bb328b1395b2ccf2f'
            'current' => 'e7140c092ecf6ed3401f922efc81b635cb253d90f56cad5ad9765511edf6b2d8c663fcb5648617d561ecd2cf7ae8503d7d2ce3b88f46844d5771909aa0dba188'
        )
        '.user.ini' => Array &3 (
            'expected' => '467d4f028c447895716a2b7859ed6e569f8ee34b87b51d73dab2e6a9ca4fbe63172c7be3e365ae864a60408286afcce852dd38ee544b61685ed4ee5e021fecb0'
            'current' => '8362e2f91924dde144162fe6b6757d88b8e6cd7266fd0e90e32aee37e662b7854384a211ac44e8703fa093833df6d5cceef5426eb141dc2638ff88d245cea3c0'
        )
    )
) is identical to Array &0 ().

/drone/src/github.com/nextcloud/server/tests/lib/IntegrityCheck/CheckerTest.php:763

```
</details>


 * DB=NODB, ENABLE_REDIS=false, PHP=7.3
<details><summary>Show full log</summary>

```
There was 1 error:

1) Test\IntegrityCheck\CheckerTest::testWriteCoreSignatureWithValidModifiedHtaccessAndUserIni
Exception: /drone/src/github.com/nextcloud/server/tests/data/integritycheck/htaccessWithValidModifiedContent/core is not writable

/drone/src/github.com/nextcloud/server/lib/private/IntegrityCheck/Checker.php:283
/drone/src/github.com/nextcloud/server/tests/lib/IntegrityCheck/CheckerTest.php:660

--

There was 1 failure:

1) Test\IntegrityCheck\CheckerTest::testVerifyCoreSignatureWithValidModifiedHtaccessAndUserIniSignatureData
Failed asserting that Array &0 (
    'INVALID_HASH' => Array &1 (
        '.htaccess' => Array &2 (
            'expected' => '9a37a508ad4cc3a7ff668e3cd63177891e5601143fa18bb605346020d5f3cd7979748beab956554aca43ae59ff146755cfe83de0f93a3a7bb328b1395b2ccf2f'
            'current' => 'e7140c092ecf6ed3401f922efc81b635cb253d90f56cad5ad9765511edf6b2d8c663fcb5648617d561ecd2cf7ae8503d7d2ce3b88f46844d5771909aa0dba188'
        )
        '.user.ini' => Array &3 (
            'expected' => '467d4f028c447895716a2b7859ed6e569f8ee34b87b51d73dab2e6a9ca4fbe63172c7be3e365ae864a60408286afcce852dd38ee544b61685ed4ee5e021fecb0'
            'current' => '8362e2f91924dde144162fe6b6757d88b8e6cd7266fd0e90e32aee37e662b7854384a211ac44e8703fa093833df6d5cceef5426eb141dc2638ff88d245cea3c0'
        )
    )
) is identical to Array &0 ().

/drone/src/github.com/nextcloud/server/tests/lib/IntegrityCheck/CheckerTest.php:763

```
</details>


 * ENABLE_OPENLDAP=true, ENABLE_REDIS=true, TESTS=integration-ldap-openldap-features
   * git clone failure
 * ENABLE_OPENLDAP=true, ENABLE_REDIS=true, TESTS=integration-ldap-openldap-uid-features
   * git clone failure

**Review by rullzer — APPROVED** [review]
Fine by me :+1:

**Comment by MorrisJobke** [comment]
unit tests still fail - let me look into this.

File: tests/data/integritycheck/htaccessWithValidModifiedContent/core/signature.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
+}
```
Comment (MorrisJobke) [inline tests/data/integritycheck/htaccessWithValidModifiedContent/core/signature.json]: 🙈 I will remove this one.

**Comment by MorrisJobke** [comment]
I will update this once #14493 is in to avoid doubled work due to conflicting lines.

**Comment by MorrisJobke** [comment]
I rebased this and fixed the failing tests.

**Comment by MorrisJobke** [comment]
Failures are unrelated -> merging.

**Comment by rcdailey** [comment]
Thanks to everyone for the help on this one. You're all awesome! 🎊

**Pull request merged**